esbuild.server.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const dotenv = require('dotenv');
+
+// Pulls in .env and populates the define object.
+// This is used to populate environment variables at built time.
+dotenv.config();
+const define = {};
+for (const k in process.env) {
```

Comment:
I'm of the school of thought that says variable names should be meaningful. I know you often see variables given one letter names to shorten the code and the argument is that's it's obvious here, but I think it's better to have `key` here. It just reads nicer and saves my brain at least the one tick it takes to work out what k stands for.

Reply:
definitely, thanks for picking up on that, will change